addons/l10n_ec/models/account_chart_template.py

```diff
@@ -50,6 +51,14 @@ def _l10n_ec_configure_ecuadorian_tax_groups_accounts(self, companies):
                 self.env.ref(f'l10n_ec.{tax_group_xml_id}').with_company(company).property_tax_payable_account_id = payable_account_id
                 self.env.ref(f'l10n_ec.{tax_group_xml_id}').with_company(company).property_tax_receivable_account_id = receivable_account_id
 
+    def _l10n_ec_configure_default_company_taxes(self, companies):
+        # From 01 april, the default taxes for sale and purchase are 15%
+        for company in companies.filtered(lambda company: company.account_fiscal_country_id.code == 'EC' and
+                                                              company.chart_template_id == self.env.ref('l10n_ec.l10n_ec_ifrs')):
+            # Default tax for new companies will change periodically, better to have a method than to re-write all xml and csv changing the tax sequences
+            company.account_sale_tax_id = self.env.ref(f'l10n_ec.{company.id}_tax_vat_15_411_goods')
+            company.account_purchase_tax_id = self.env.ref(f'l10n_ec.{company.id}_tax_vat_15_510_sup_01')
+
```

Comment:
@stevTresCloud Let's avoid this and maybe in the XML, we just put the default taxes first in the file (or with the sequence)  The one with the first sequence is normally set as the first here already.

Reply:
Hi @jco-odoo. Ok, I changed the sequence of the new taxes, and it fixed the problem. So it's already done and I proceed to delete this method.